images/virtualization-artifact/cmd/virtualization-controller/main.go

```diff
@@ -238,6 +237,12 @@ func main() {
 		virtualMachineIPLeasesRetentionDuration = "10m"
 	}
 
+	clusterUUID := os.Getenv(clusterUUIDEnv)
+	if clusterUUID == "" {
+		log.Error("clusterUUID not found, but required")
```

Comment:
Should be error about env:

```suggestion
		log.Error(fmt.Sprintf("Required %s environment variable is empty, should contain cluster UUID", clusterUUIDEnv)
```